pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/KubernetesRuntime.java

```diff
@@ -190,6 +190,8 @@
             case PYTHON:
                 logConfigFile = pulsarRootDir + "/conf/functions-logging/console_logging_config.ini";
                 break;
+            case GO:
+                break;
```

Comment:
is this supported? if it is not fully supported, I would suggest throwing UnsupportedOperationException in this place until we confirm it is working.